src/Card/tests/CardSection.test.jsx

```diff
@@ -54,18 +54,4 @@ describe('<CardSection />', () => {
     expect(wrapper.exists('.pgn__card-section-loader')).toBe(true);
     expect(wrapper.props().isLoading).toBe(true);
   });
-  it('renders muted variant', () => {
```

Comment:
[curious] Should we leave this test in considering we're leaving the `muted` prop for `Card.Section`?